exercises/crypto-square/src/CryptoSquare.hs

```diff
@@ -1,4 +1,4 @@
 module CryptoSquare (encode) where
 
 encode :: String -> String
-encode = error "You need to implement this function."
+encode xs = error "You need to implement this function."
```

Comment:
sorry that I know I'm contradicting https://github.com/exercism/xhaskell/pull/504#discussion_r105556991 here, but it seemed a bit inconsistent here to use `xs` but `phrase` in `atbash-cipher`?